src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,68 @@
+.footer {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  background-color: #0F1121;
+  font-family: "Montserrat", sans-serif;
```

Comment:
Where you find Montserrat on design(git pull origin master and in fonts you can find Mont font)